Makefile

```diff
@@ -353,9 +353,20 @@ lock-package-images: check-carvel # Updates the image lock file for a package. U
 	@printf "\n===> Updating image lockfile for package $${PACKAGE}/$${VERSION}\n";\
 	cd addons/packages/$${PACKAGE}/$${VERSION} && kbld --file bundle --imgpkg-lock-output bundle/.imgpkg/images.yml >> /dev/null;\
 
-push-package: check-carvel # Build and push a package template. Tag will default to `latest`. Usage: make push-package PACKAGE=foobar VERSION=1.0.0
+push-package: check-carvel # Verify openAPIv3 schema in package before build and push a package template. Tag will default to `latest`. Usage: make push-package PACKAGE=foobar VERSION=1.0.0
 	@printf "\n===> pushing $${PACKAGE}/$${VERSION}\n";\
-	cd addons/packages/$${PACKAGE}/$${VERSION} && imgpkg push --bundle $(OCI_REGISTRY)/$${PACKAGE}:$${VERSION} --file bundle/;\
+	./hack/packages/verify-openapischema-for-package.sh $(PACKAGE) $(VERSION) \
+	&& cd addons/packages/$${PACKAGE}/$${VERSION} && imgpkg push --bundle $(OCI_REGISTRY)/$${PACKAGE}:$${VERSION} --file bundle/;\
+
+generate-openapischema-package: #Generate package with OpenAPI v3 schema
+	@printf "\n===> generating OpenAPIv3 schema for $${PACKAGE}/$${VERSION}\n";\
+	./hack/packages/check-sample-values-and-render-ytt.sh $(PACKAGE) $(VERSION) \
+	&& cd addons/packages/$${PACKAGE}/$${VERSION}/bundle/config \
+	&& ytt -f schema.yaml --data-values-schema-inspect -o openapi-v3 > openapi-schema.yaml \
```

Comment:
Can we put this `openapi-schema.yaml` in `artifacts/<package_name>/<version>` as well?

Reply:
Fixed